src/control/common/proto/ctl/ctl.pb.go

```diff
@@ -7,7 +7,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.26.0
-// 	protoc        v3.6.1
```

Comment:
nit: We generally avoid committing these version-only changes to the generated files in order to cut down on churn and review burden. I'm fine with leaving these as-is though because hopefully everyone is on the same version of the tools now.